### DISCUSSION TITLE
Stuck at checking prerequisites...

### DISCUSSION THREAD

**Discussion by henricast** [Romanitho/Winget-AutoUpdate#784] [Q&A]
Good day!
Im trying to deploy the WAU via AD GPO to my domain users in my company but I'm stuck.
I've added the .msi installer in _Computer Configuration > Policies > Software Settings > Software installer_ and I've managed to install on the test computer. But, when I restart the pc or do Logon, I expect that the WAU start upgrading my apps, but this won't happen. I see the WAU starts on log file, but it always stops on **Checking prerequisites** ... I see there are task schedulers with NT SYSTEM , SO I think this user has any block on pc...
I've already tried to enable GPO management and enabling the option _"User context execution"_ and _"Activate WAU GPO managemenet"_ but no success too...
I have more than 600 computers to deploy, in different country regions, so manual Install is not viable.
How do I resolve this requisites? How can I enable WAU to upgrade the apps of my clients?
help! 
last log:

#################################################################
#    04/12/2024 - CHECK FOR APP UPDATES (System context - Connected user)
#################################################################
10:53:33 - WAU Policies management Enabled.
10:53:33 - Notification Level: SuccessOnly. Notification Language: Português
10:53:33 - Checking internet connection...
10:53:33 - Connected !
10:53:33 - Checking prerequisites...

**Comment by KnifMelti** [comment]
I don't know what's wrong, but you can find out what's supposed to happen in:
https://github.com/Romanitho/Winget-AutoUpdate/blob/f4bcca43c3f14b186bfc4a5ce9121e67759c12a9/Sources/Winget-AutoUpdate/functions/Install-Prerequisites.ps1
The fact that nothing more is written in the log does not make sense, there's a bunch of writing to the log in that function...
NT SYSTEM is the computer account allowing  everything to be done.
